build-system/tasks/dep-check.js

```diff
@@ -75,7 +75,7 @@ function Rule(config) {
   /** @private @const {!GlobsDef} */
   this.mustNotDependOn_ = toArrayOrDefault(config.mustNotDependOn, []);
 
-  /** @private @const {!GlobsDef} */
+  /** @private @const {!Array<string>} */
```

Comment:
probably no need to change type here since `GlobsDef` is a typealias to `!Array<!GlobDef>` and `GlobDef` is in turn a typealias to `string` which makes them equivalent. but it makes the input string have a bit more context, sorta like doing a type alias for ```js/** @typedef {string} */ var Url;```.

I dont know if this is done or valid in closure compiler annotations but we usually did this in haskell/scala. (don't feel strongly either way of course)